### PR TITLE
Buildx for creating ARM images

### LINKED ISSUE #146: ARM docker image in CI
We should add a build for an ARM docker image to the CI so that future ARM based machines (M1, Raspberry Pi's) can also run the docker image natively.

### PR DESCRIPTION
**Pull request by KiOui**
Added `buildx` to the CI for creating ARM docker images.

Closes #146

### CODE REVIEW

**Comment by KiOui** [comment]
Also changed `tosti-production` back to `tosti` for the image name as that is more nice when pulling the image.

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -7,7 +7,8 @@ on:
       - master
 
 env:
-  DOCKER_IMAGE: "docker.pkg.github.com/kioui/tosti/tosti-production:latest"
+  DOCKER_IMAGE: docker.pkg.github.com/kioui/tosti/tosti:latest
+  DOCKER_TAG_PRODUCTION: docker.pkg.github.com/kioui/tosti/tosti:${{ github.sha }}
```
Comment (JobDoesburg) [inline .github/workflows/docker.yaml:11]: If you're doing this, then probably you would just want to build a docker on every PR (with the hash as label), and build :latest on push to master.

Comment (KiOui) [reply .github/workflows/docker.yaml:11]: Yeah I thought this would be nice for downloading "older" images. If you don't agree we can remove it.

Comment (JobDoesburg) [reply .github/workflows/docker.yaml:11]: I don't have a problem with building dockers for each PR! But this action only runs on pushes to master, so I don't see what this does

Comment (JobDoesburg) [reply .github/workflows/docker.yaml:11]: Maybe we should tag `:stable` on release, `:latest` on push to master and `:<commithash>` on each PR?

Comment (KiOui) [reply .github/workflows/docker.yaml:11]: I made up my mind, I'll remove the commit hash push but let's use the stable and latest tags. I think that makes it pretty nice.

**Comment by JobDoesburg** [comment]
So for stable and latest we still need to setup CI

**Comment by KiOui** [comment]
> So for stable and latest we still need to setup CI

Yes

**Comment by KiOui** [comment]
Not for latest though.

**Review by JobDoesburg — APPROVED** [review]

**Pull request merged**